src/lib/util.ts

```diff
@@ -77,6 +77,16 @@ export async function closeConnection(uuid: string) {
 	}
 }
 
+export function transformDiscordLocale(discordLocale: string): string {
+	if (discordLocale === "uk") discordLocale = "ua"
+	const localeWithCountryCode = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a, b) => (b ? a + b.toLowerCase() : a)),
```

Comment:
Surely eslint would scream at this, right? If not, I will
```suggestion
	const localeWithCountryCode = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a, b) => (b ? `${a}${b.toLowerCase()}` : a)),
```

Reply:
It doesn't scream at this 👍🏻

Reply:
Well then I do